src/main/webapp/js/instructor.js

```diff
@@ -305,6 +305,42 @@ function bindStudentPhotoHoverLink(elements) {
     });
 }
 
+function bindDeleteButton() {
```

Comment:
`bind*Buttons()` because this function (and similar functions) binds to all buttons of the same type, not just one.